exercises/isogram/src/test/java/IsogramTest.java

```diff
@@ -0,0 +1,69 @@
+package test;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+import exemple.Isogram;
+
+public class IsogramTest {
+
+	@Test
+	public void testIsogram() {
+		Isogram iso = new Isogram("duplicates");
+		assertTrue(iso.isogramChecker());
+	}
+	
+	@Test
```

Comment:
You'll want to annotate all tests except the first one with `@Ignore`, so the users can explore their solution using each test as a goalpost.